patches/minecraft/net/minecraft/entity/player/ServerPlayerEntity.java.patch

```diff
@@ -8,7 +8,26 @@
        boolean flag = this.field_70170_p.func_82736_K().func_223586_b(GameRules.field_223609_l);
        if (flag) {
           ITextComponent itextcomponent = this.func_110142_aN().func_151521_b();
-@@ -614,13 +615,14 @@
+@@ -603,9 +604,16 @@
+    }
+ 
+    @Nullable
++   @Deprecated // Forge: Use ITeleporter safe version
+    protected PortalInfo func_241829_a(ServerWorld p_241829_1_) {
+-      PortalInfo portalinfo = super.func_241829_a(p_241829_1_);
+-      if (portalinfo != null && this.field_70170_p.func_234923_W_() == World.field_234918_g_ && p_241829_1_.func_234923_W_() == World.field_234920_i_) {
++      return this.getPortalInfo(p_241829_1_, p_241829_1_.func_85176_s());
++   }
++   
++   @Nullable
++   protected PortalInfo getPortalInfo(ServerWorld p_241829_1_, net.minecraftforge.common.util.ITeleporter teleporter) {
```

Comment:
Same reason as Entity.java.patch. It technically could be null and I could add extra checks in this case, but teleporter really should never be null in the first place since that would cause many logic issues in game.